src/Microsoft.AspNet.Session/DistributedSession.cs

```diff
@@ -14,6 +14,9 @@
 
 namespace Microsoft.AspNet.Session
 {
+    /// <summary>
+    /// Represent a distributed cache session.".
```

Comment:
Plus I'm not sure this comment really says much about what it is. It's just kind of saying the type name.